recipes/rawpy/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "rawpy" %}
+{% set version = "0.24.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/letmaik/rawpy/archive/v{{ version }}.tar.gz
+  sha256: d41b41332b51096fc2ccc451eea8547e28fbfd992c277dfdbfb31eb8dab913fe
+  patches:
+    - patches/0001-Fix-include-path.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - cython
```

Comment:
This should not be here. If necessary, then there should be also the other cross compilation stuff.